doc/references.bib

```diff
@@ -1266,3 +1266,49 @@ @article{smith2009threshold
 	url={https://doi.org/10.1016/j.neuroimage.2008.03.061},
 	doi={10.1016/j.neuroimage.2008.03.061}
 }
+
+@book{Montgomery,
+  author = {Montgomery, Douglas C. and Peck, Elizabeth A. and Vining, Geoffrey G.},
+  isbn = {0471754951},
+  keywords = {montgomery},
+  publisher = {Wiley \& Sons},
+  title = {Introduction to Linear Regression Analysis (4th ed.)},
+  year = 2006
+}
+
+@book{GreeneEconometric,
```

Comment:
This is totally unrelated to this PR, but I think we should restore the alphabetical order of references that got broken a while ago. I'm afraid that contributors will duplicate entries if this stays unsorted...

Reply:
Agreed. I can open a separate issue on that.